pymc3/distributions/continuous.py

```diff
@@ -2642,11 +2681,16 @@ def logcdf(self, value):
         """
         alpha = self.alpha
         beta = self.beta
+        # To avoid gammaincc C-assertion when given invalid values (#4340)
```

Comment:
"gammainc" or "gammaincc"?

Reply:
the regularized _upper_ gamma function, also known as gammaincc or gamma Q.

Reply:
The convention is "inc" for incomplete and "c" for "complementary".
See here for the corresponding SciPy function:
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.gammaincc.html

Reply:
@domenzain  is right. The two methods call a different function `gammainc` vs `gammaincc`. Not a typo ;)